### PR TITLE
doc(adt): use a full CRUD snippet for all involved operations

### PR DESCRIPTION
**Pull request by drwill-ms**

### CODE REVIEW

File: sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md [hunk 1]
```diff
@@ -111,23 +111,26 @@ catch (Exception ex)
 }
 ```
```
Comment (drwill-ms) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md:113]: This file is easier to look at just in [readme form](https://github.com/Azure/azure-sdk-for-net/tree/feature/adt/drwill/crudSnippet/sdk/digitaltwins/Azure.DigitalTwins.Core/samples).

File: sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md [hunk 2]
```diff
@@ -111,23 +111,25 @@ catch (Exception ex)
...
 
-## Create and delete digital twins
+## Create, get, list, update, and delete digital twins and update components
```
Comment (bikamani) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md:114]: Did you mean to add both "get" and "list"?

File: sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs [hunk 3]
```diff
@@ -76,18 +80,14 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
 
             string basicDtPayload = JsonSerializer.Serialize(basicTwin);
 
-            Response<string> createBasicDtResponse = await client.CreateDigitalTwinAsync(basicDtId, basicDtPayload);
-            Console.WriteLine($"Created digital twin with Id {basicDtId}. Response status: {createBasicDtResponse.GetRawResponse().Status}.");
-
-            #endregion Snippet:DigitalTwinsSampleCreateBasicTwin
+            Response<string> createBasicDtResponse = await client.CreateDigitalTwinAsync("basicDigitalTwin-123-Id", basicDtPayload);
```
Comment (bikamani) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs:83]: may be better to define these ids/strings within snippet and use that variable here and other places below.

**Review by bikamani — APPROVED** [review]

File: sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md (hunk 2 above)
Comment (drwill-ms) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md:114]: Yes, but now you mention it we don't do a list in this sample, so I'll remove it.

File: sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs (hunk 3 above)
Comment (drwill-ms) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs:83]: See the comment at the top of the file as to why we're using hard-coded strings throughout.

File: sdk/digitaltwins/Azure.DigitalTwins.Core/src/Serialization/UpdateOperationsUtility.cs [hunk 4]
```diff
@@ -13,15 +13,118 @@ namespace Azure.DigitalTwins.Core.Serialization
     /// For more samples, see <see href="https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/digitaltwins/Azure.DigitalTwins.Core/samples">our repo samples</see>.
     /// </remarks>
     /// <example>
-    /// <code snippet="Snippet:DigitalTwinsSampleUpdateComponent">
+    /// <code snippet="Snippet:DigitalTwinsSampleDigitalTwinsLifecycle">
```
[the remark this answers is not in the document]
Comment (drwill-ms) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/src/Serialization/UpdateOperationsUtility.cs:16]: That is how we get them into the web documentation.

This PR is to show the alternate option. It is not a slam dunk for us to take this.

**Pull request closed without merging**